roles/maintenance/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
this role is already a dependency of the ec2-app role already, so isn't needed here.